WatchGuard/watchguard-firebox/ingest/parser.yml

```diff
@@ -92,7 +92,7 @@ stages:
           watchguard.firebox.network.tcp.sequence: '{{parsed_event.message.tcp_seq}}'
           watchguard.firebox.network.tcp.window: '{{parsed_event.message.tcp_window}}'
           watchguard.firebox.network.duration: '{{parsed_event.message.duration}}'
-          watchguard.firebox.application.duration: '{{parsed_event.message.elapsed_time}}'
+          watchguard.firebox.application.duration: '{{parsed_event.message.elapsed_time.split(" ")[0]}}'
```

Comment:
Maybe add a filter to ensure that `elapsed_time` is not None. What do you think about?

Reply:
Done ! :sun_with_face: